dist/index.d.ts

```diff
@@ -722,7 +721,7 @@ declare const BOOSTS: {
             damage: 0.3;
         };
     };
-};
+} & Record<BodyPartConstant, Record<MineralBoostConstant, Record<BoostModifier, number>>>;
```

Comment:
Not all minerals nor all modifiers are listed.
Exemple: `boost.tough.H` and `boost.tough.GO.fatigue` are undefined

```suggestion
} & Record<BodyPartConstant, Partial<Record<MineralBoostConstant, Partial<Record<BoostModifier, number>>>>>;
```

Reply:
I just made the `| undefined` explicit, because `Partial` wasn't cutting it. There's two added tests that shows what auto-completion does:
```ts
// Accessing a valid BOOST is fine
BOOSTS.attack.UH2O.attack;

// Accessing something that might not be valid adds optionals
BOOSTS.claim?.GHO2?.capacity;
```

Reply:
Actually, I had to use that in a TS mock, and it ends up not quite working soooo…